workspaces/client/app/routes/dashboard/oauth-apps/register/page.tsx

```diff
@@ -185,7 +302,7 @@ export default function Register() {
 						<ButtonLike variant="secondary">キャンセル</ButtonLike>
 					</Link>
 					<button type="submit" disabled={isPending}>
-						<ButtonLike disabled={isPending}>Save</ButtonLike>
+						<ButtonLike disabled={isPending}>登録する</ButtonLike>
```

Comment:
```suggestion
						<ButtonLike disabled={isPending}>登録</ButtonLike>
```
かなぁ

- キャンセル <-> 登録
- キャンセルする <-> 登録する

細かいけど